homeassistant/components/alexa_devices/sensor.py

```diff
@@ -0,0 +1,88 @@
+"""Support for sensors."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Final, cast
+
+from aioamazondevices.api import AmazonDevice
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.const import LIGHT_LUX, UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from .coordinator import AmazonConfigEntry
+from .entity import AmazonEntity
+
+# Coordinator is used to centralize the data updates
+PARALLEL_UPDATES = 0
+
+
+@dataclass(frozen=True, kw_only=True)
+class AmazonSensorEntityDescription(SensorEntityDescription):
+    """Amazon Devices sensor entity description."""
+
+    converted_unit: Callable[[AmazonDevice, str], str] | None = None
```

Comment:
```suggestion
    native_unit_of_measurement_fn: Callable[[AmazonDevice, str], str] | None = None
```
I think this is more clear than `converted_unit`